config/conf.json

```diff
@@ -68,7 +68,7 @@
                     "scikit-learn"
                 ]
             },
-            "version" : "1.0.9",
+            "version" : "1.0.11",
```

Comment:
I'm going to skip `1.0.10`, since I don't know if reusing that ID is going to cause issues (I'm guessing some of these built behind the scenes)

Reply:
I could also just remove these bad tags in GCP...lmk

Reply:
Let me know if you prefer that - my default would be to keep the PR as-is so I don't need to rerun the tests again (and avoiding reuse seems slightly safer).

Reply:
@n8landolt do you have a preference? basically in your new PR, if we don't remove the old tags in GCP, then you'll need to bump all image versions again...

Reply:
I think we want to just bump it to .11 No real harm in missing a minor version, right?